cmd/scollector/collectors/interval.go

```diff
@@ -48,10 +49,15 @@ func (c *IntervalCollector) Run(dpchan chan<- *opentsdb.DataPoint) {
 		}
 		next := time.After(interval)
 		if c.Enabled() {
+			timeStart := time.Now()
 			md, err := c.F()
+			timeFinish := time.Now()
+			result := "success"
 			if err != nil {
 				slog.Errorf("%v: %v", c.Name(), err)
+				result = "error"
 			}
+			Add(&md, "scollector.collect.duration", float64(timeFinish.Sub(timeStart))/float64(time.Millisecond), opentsdb.TagSet{"collector": c.Name(), "result": result}, metadata.Gauge, metadata.MilliSecond, "Duration in milliseconds and result of collector runs in each interval.")
```

Comment:
Change name to `scollector.collector.duration`. The `scollector.collect` namespace is reserved for the collect package, which this isn't. Remove result from the tag, and use `timeFinish.Seconds()` as the value, and fix the description string to list seconds as the unit.